crates/bevy_hierarchy/src/hierarchy.rs

```diff
@@ -9,17 +9,19 @@ use bevy_utils::tracing::debug;
 /// Despawns the given entity and all its children recursively
 #[derive(Debug)]
 pub struct DespawnRecursive {
-    entity: Entity,
+    /// Target entity
+    pub entity: Entity,
 }
 
 /// Despawns the given entity's children recursively
 #[derive(Debug)]
 pub struct DespawnChildrenRecursive {
-    entity: Entity,
+    /// Target entity
+    pub entity: Entity,
 }
 
 /// Function for despawning an entity and all its children
-pub fn despawn_with_children_recursive(world: &mut World, entity: Entity) {
+fn despawn_with_children_recursive(world: &mut World, entity: Entity) {
```

Comment:
I know I approved it already, but I just noticed it. Why is this not pub anymore? It might make sense to do it, but I'm not sure it should be in a PR that is just adding logs

Reply:
too late you approved!

I made it private to have the guarantee that reaching here only happens through the command, and to make it similar to the other despawn_children function

Reply:
This should be noted in a migration guide section.

Reply:
remove that change to not be an breaking issue for now